bundles/CoreBundle/src/Command/SearchBackendReindexCommand.php

```diff
@@ -64,6 +75,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     DataObject::OBJECT_TYPE_VARIANT,
                 ]);
             }
+            // Load the data that has already been processed
+            if (!$isFull && $ids = $db->fetchFirstColumn("SELECT id FROM search_backend_data WHERE maintype = '". $type ."' ")) {
```

Comment:
Using this query without a limit could potentially lead to memory issues, I'd suggest to do them in chunks/batches?